sticht/slack.py

```diff
@@ -176,16 +203,23 @@ def get_summary_blocks_for_deployment(self) -> List[Dict]:
                     'elements': button_elements,
                 },
             )
+
+        # If blocks is going to be longer than 3kB, trim summary_parts. This should help avoid invalid_blocks error.
+        json_blocks = json.dumps(blocks)
+        if len(json_blocks) > 2900:
+            excess_length = len(json_blocks) - 2900
+            blocks[0]['text']['text'] = blocks[0]['text']['text'][:-excess_length]
+
```

Comment:
This is the actual meat of the change. The rest of this is type annotations so that mypy knows what `blocks[0]['text']['text']` actually is. (Otherwise, we get errors like https://gitter.im/python/typing?at=5e5ecdd9cb91b5224f0b4889)